static/app/components/segmentedControl.spec.tsx

```diff
@@ -0,0 +1,111 @@
+import {render, screen, userEvent} from 'sentry-test/reactTestingLibrary';
+
+import {SegmentedControl} from 'sentry/components/segmentedControl';
+
+describe('SegmentedControl', function () {
+  it('renders with uncontrolled value', function () {
```

Comment:
Could we also test that the `onChange` is triggered when it's uncontrolled with a `jest.fn` isCalled check?